README.md

```diff
@@ -93,6 +93,40 @@ Explainations for the dependencies:
 4. close the current terminal.
 
 
+### 3.1 Install ZED-ROS-Wrapper ##
+-------------------------
+[Official documentation](https://www.stereolabs.com/docs/ros/)
+
+1. Enter the catkin workspace:
+	```bash
+	 cd ~/manta_ws/src
+	```
+
+2. Clone the zed-ros-wrapper repo:
+	```bash
+	 git clone https://github.com/stereolabs/zed-ros-wrapper.git
+	```
+
+3. Jump up to parent directory:
+	```bash
+	 cd ../
+	```
+
+4. Install dependencies:
+	```bash
+	 rosdep install --from-paths src --ignore-src -r -y
```

Comment:
Building after just doing this gives the following error:

```
CMake Error at /home/chstr/vortex_ws/src/zed-ros-wrapper/zed_nodelets/CMakeLists.txt:20 (message):

   ZED SDK v3.x not found, install it from:
   https://www.stereolabs.com/developers/
```

The error message is obvious enough to fix without any guide, but since this should be a complete guide, I think there should be a step in the very beginning explaining how to install the ZED SDK. A TLDR of https://www.stereolabs.com/docs/installation/linux/ should do :+1: